client/write.go

```diff
@@ -96,7 +95,6 @@ type writeState struct {
 	session           *session
 	topoMap           topology.Map
 	op                *writeOp
-	ctx               context.Context
```

Comment:
NB: can remove this as the writeState is no longer responsible for the lifecycle of the nsID+tsID, it's responsibility is pushed up to the user of the client/Session API

Reply:
Yeah so, issue is sometimes `Write(..)` returns before RPCs to all replicas have finished (quorum writes return early once quorum is reached).

Reply:
Nice catch, didn't think of that. Will address in next patch.